### PR TITLE
new licensed health care facilities page

### PR DESCRIPTION
**Pull request by ZachBeck**
New licensed health care facilities that will replace existing health care facilities.

### CODE REVIEW

File: src/pages/products/sgid/health/licensed-health-care-facilities.astro [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <UpdateHistory slot="update-history" updates={page.updateHistory} />
+
+  <Section title="More resources" slot="more-resources" titlePosition="center">
+    <GridForMoreResources>
+      <CardWithPopularLink
+        title="DHHS Facility Information"
```
Comment (steveoh) [inline src/pages/products/sgid/health/licensed-health-care-facilities.astro:60]: sentence case
```suggestion
        title="DHHS facility information"
```

**Review by steveoh — COMMENTED** [review]
looks good

**Comment by steveoh** [comment]
I assume we need to wait to merge this until some porter stuff happens?

**Comment by ZachBeck** [comment]
I just started the deprecation process today. Maybe we merge this and take down the existing page next week.

**Comment by steveoh** [comment]
refs https://github.com/agrc/porter/issues/257

**Review by steveoh — APPROVED** [review]

**Pull request merged**